zipkin-cassandra/src/test/scala/com/twitter/zipkin/cassandra/CassandraSpanStoreFactorySpec.scala

```diff
@@ -46,4 +46,38 @@ class CassandraSpanStoreFactorySpec extends FunSuite with Matchers {
       asList(new InetSocketAddress("1.1.1.1", 9143), new InetSocketAddress("2.2.2.2", 9042))
     )
   }
+
+  test("creatingClusterBuilder with authentication") {
+    TestFactory.nonExitingMain(Array(
+      "-zipkin.store.cassandra.auth", "true",
+      "-zipkin.store.cassandra.user", "user",
+      "-zipkin.store.cassandra.password", "pass"
+    ))
+
+    val builder = TestFactory.createClusterBuilder()
+    val authProvider     = get_privates(builder, "authProvider")
+    val user = get_privates(authProvider, "username")
+    val pass = get_privates(authProvider, "password")
+    user should be("user")
+    pass should be("pass")
+  }
+
+  test("creatingClusterBuilder without authentication") {
+    TestFactory.nonExitingMain(Array(
+      "-zipkin.store.cassandra.auth", "false"
+    ))
+
+    val builder = TestFactory.createClusterBuilder()
+    val authProvider     = get_privates(builder, "authProvider")
+    val user = get_privates(authProvider, "username")
```

Comment:
@adriancole actually I couldn't find it, am I doing something different?
![screen shot 2015-08-19 at 7 50 21 pm](https://cloud.githubusercontent.com/assets/473680/9374555/10bf8a76-46ac-11e5-835e-07869c118c8f.png)